src/diffusers/loaders/lora.py

```diff
@@ -883,11 +907,11 @@ def save_lora_weights(
         safe_serialization: bool = True,
     ):
         r"""
-        Save the LoRA parameters corresponding to the UNet and text encoder.
+        Save the UNet and text encoder LoRA parameters.
```

Comment:
and here too please 🙏

Reply:
https://github.com/huggingface/diffusers/blob/c697f524761abd2314c030221a3ad2f7791eab4e/examples/dreambooth/train_dreambooth_lora_sdxl.py#L819